### PR TITLE
Fixes #25: Fixing Bootslide for RTL

### LINKED ISSUE #25: Bootslide slides positioned incorrectly for RTL
Noticed this while looking at [this issue in Scoreboard](https://github.com/SpiderStrategies/Scoreboard/issues/54676)

Issue: The slides for the bootslide in RTL are overflowing and overlapping each other.

Examples: 
Screenshot 2023-09-27 at 12 21 09 PM
Screenshot 2023-09-27 at 12 20 43 PM

### PR DESCRIPTION
**Pull request by malik-habib**
Changing the .slide and .resetMargins to check for RTL

### CODE REVIEW

File: src/bootslide.js [hunk 1]
```diff
@@ -72,17 +72,20 @@ Bootslide.prototype.reset = function () {
...
 
+    var translateValue = self.isRtl() ? prevWidth * -1 : prevWidth
+
     // Store the point that bootslide-menu-slider has to translate to
     // to show this section
-    $(this).data('target-translate', prevWidth * -1)
+    $(this).data('target-translate', translateValue)
 
-    return 'translate(' + prevWidth + 'px,0)'
+    return 'translate(' + translateValue + 'px,0)'
```
Comment (mattsgarlata) [inline src/bootslide.js:88]: Previously the value here and the value two lines up were not the same. Are you sure it’s correct for them to now be the same?

Comment (malik-habib) [reply src/bootslide.js:88]: I think this was functioning similarly since I added the `* -1` to the `.slide` function, but I changed this back now.

Comment (nathanbowser) [reply src/bootslide.js:88]: This is being a little nit-picky, so I apologize if it's annoying. But we're trying to now use modern JS when updating files. So when creating new variables, let's roll w/ either `const` or `let` depending what's more appropriate. `var` is leaky.

Comment (malik-habib) [reply src/bootslide.js:89]: I went ahead and made this change

File: src/bootslide.js [hunk 2]
```diff
@@ -72,17 +72,21 @@ Bootslide.prototype.reset = function () {
...
 Bootslide.prototype.resetMargins = function (ctx) {
+  const self = this
   $('.bootslide-section', ctx).css('transform', function (i) {
     // Figure out width of all previous sections (ie sections to the left)
     var prevWidth = $(this).prevAll()
       .map(function () { return $(this).width() }).get()
       .reduce(function (sum, current) { return sum + current }, 0)
 
+    const translateValue = self.isRtl() ? prevWidth * -1 : prevWidth
+    console.log(`the self is ${self} and the translateValue is ${translateValue}`)
```
Comment (nathanbowser) [inline src/bootslide.js:83]: Forgot about this logging statement

Comment (malik-habib) [reply src/bootslide.js]: Removed

**Comment by nathanbowser** [comment]
Thanks @malik-habib !

**Pull request merged**